website/docs/r/nsxt_security_group.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_security_group"
+sidebar_current: "docs-vcd-resource-nsxt-security-group"
+description: |-
+Provides a resource to manage NSX-T Security Group. Security groups are groups of data center
+group networks to which distributed firewall rules apply. Grouping networks helps you to reduce
+the total number of distributed firewall rules to be created.
+---
+
+# vcd\_nsxt\_security\_group
+
+Provides a resource to manage NSX-T Security Group. Security groups are groups of data center group
+networks to which distributed firewall rules apply. Grouping networks helps you to reduce the total
+number of distributed firewall rules to be created.
+
+Supported in provider *v3.3+* and VCD 10.1+ with NSX-T backed VDCs.
+
+## Specific usage notes
+
+## Example Usage 1 (Security Group with member networks)
+
+```hcl
+resource "vcd_nsxt_security_group" "frontend-servers" {
+  org  = "my-org"
+  vdc  = "my-nsxt-vdc"
+
+  # refering to a datasource for existing NSX-T Edge Gateway
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+
+  name = "frontend-servers"
+  description = "Security group consisting of frontend servers"
```

Comment:
Why it says frontend servers, while in practice it is linking to a network?

Reply:
I can change, but security groups are quite often created "by purpose". So when you open a firewall rule list you see something similar to:
```
Open Port TCP/443 to "Frontend servers"
Open Port TCP/3306 from "Frontend servers" to "Database servers"
```

Reply:
I'm just thinking about new users. Maybe amend description like this, so it's clear for everyone?
`description = "Security group for a network connecting the frontend servers"`

Reply:
No objections - changed.